### DISCUSSION TITLE
Standup and daylight savings

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/keda#2773] [General] [answered]
Timezones are always a pain, and recent daylight savings was another confusing moment.

Our community page says that we follow US daylight savings but I'm not sure if that is correct @zroubalik  @JorTurFer? What time did you join this week?

If it's not correct, we should update our community docs on this maybe.

_PS: Looks like some people were still joining our old link so we might need to make some more buzz about our recent change_

**Comment by JorTurFer** [comment, marked as answer]
I joined as always, 17h00 CEST

**Reply by tomkerkhove** [reply]
In that case we should update " Bi-weekly Tues 16:00 UTC (does follow US daylight savings)" from keda.sh if @zroubalik agrees, update our Google calendar and announce this on Slack & Twitter to clarify.

**Reply by JatinSanghvi** [reply]
Can the time be specified in say PDT if it follows US daylight saving time? UTC itself does not follow US daylight saving, so it would be either "16:00 UTC on winters only" or "16:00 UTC for rest of the year only" but the sentence does not disambiguate between the two.

**Reply by JorTurFer** [reply]
Maybe we could say, 5PM CEST. That's a specific notation during all the year :)

**Reply by zroubalik** [reply]
@JorTurFer agree.

Timezones are mess 💩

**Reply by JatinSanghvi** [reply]
Sounds good. Just a comment, Daylight saving has already begun in US, but it's two more weeks to go for Europe. In that sense, we are not actually following US daylight savings?

**Reply by v-shenoy** [reply]
Can the [README](https://github.com/kedacore/keda#community) be updated as well?

**Reply by tomkerkhove** [reply]
Did anyone update the website and README to follow-up on this?

**Reply by zroubalik** [reply]
I dont

**Reply by JorTurFer** [reply]
Me neither

**Reply by tomkerkhove** [reply]
PRs opened to improve docs that we are doing standups at 5PM EU time (CET/CEST).
